utils/vectorscan/Makefile

```diff
@@ -0,0 +1,78 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=vectorscan
+PKG_VERSION:=5.4.9
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/VectorCamp/vectorscan/tar.gz/$(PKG_NAME)/$(PKG_VERSION)?
+PKG_HASH:=e61c78f26a9d04ccffab0df1159885c4503fc501172402c57f7357a2126ea3c6
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_NAME)-$(PKG_VERSION)
+
+PKG_MAINTAINER:=John Audia <therealgraysky@proton.me>
+PKG_LICENSE:=BSD
+PKG_LICENSE_FILES:=LICENSE
+PKG_BUILD_PARALLEL:=1
+
+HOST_BUILD_DEPENDS:=ragel/host ninja/host python/host boost/host
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/vectorscan
+  CATEGORY:=Utilities
+  SECTION:=utils
+  TITLE:=Vectorscan
+  URL:=https://github.com/VectorCamp/vectorscan
+endef
```

Comment:
I think not only because this can run on x86 as well.  We currently do not package hyperscan for x86 which is the native one whereas this is a fork that works on ARM64/ARM as well as on x86.  What do you think?

Reply:
OK... if ARM64 = `@aarch64` and if x86 = `@x86` what is the alias for arm?  I believe it will build and run for that as well.

Reply:
Done